gulpfile.js

```diff
@@ -113,7 +113,7 @@ gulp.task('set-variables-mist', function () {
 gulp.task('set-variables-wallet', function () {
     type = 'wallet';
     filenameLowercase = 'ethereum-wallet';
```

Comment:
This was used in an earlier stage of the build script. we can probably get rid of it now